activerecord/test/cases/adapters/postgresql/postgresql_adapter_test.rb

```diff
@@ -137,6 +137,32 @@ def test_exec_insert_default_values_quoted_schema_with_returning_disabled_and_no
         assert_equal expect.to_i, result.rows.first.first
       end
 
+      def test_enable_extension
```

Comment:
Would you consider to move this test to test/cases/adapters/postgresql/extension_migration_test.rb ?

Reply:
Yes! you are absolutely right.